pkg/services/featuremgmt/registry.go

```diff
@@ -1713,6 +1713,20 @@ var (
 			Stage:       FeatureStagePublicPreview,
 			Owner:       identityAccessTeam,
 		},
+		{
+			Name:        "ABTestFeatureToggleA",
+			Description: "Test feature toggle to see how cohorts could be set up AB testing",
+			Stage:       FeatureStageExperimental,
+			Owner:       grafanaSharingSquad,
+			Expression:  "false",
```

Comment:
There is a property to explicitly hide the feature toggle from docs, which is desired in this case.
```suggestion
			Expression:  "false",
			HideFromDocs: true,
```